_pages/en_US/troubleshooting.txt

```diff
@@ -276,13 +276,19 @@ If your console is not on those firmwares, it likely indicates that you already
 <summary><u>"An error has occurred, forcing the software to close..." (white message box)</u></summary>
 
 There is an issue with your `otherapp.bin` file (it is missing, misplaced, or corrupted). Download the latest release of [universal-otherapp](https://github.com/TuxSH/universal-otherapp/releases/latest) and place it on the root of your SD card.
+
+If the above doesn't fix your issue, and you are using an Old 3DS / Old 3DS XL / Old 2DS, you may be encountering a specific issue involving cartridge updates. Take another look at the system version. If the last number, e.g. 4 in 11.3.0-**4**U, is 4 and lower and the system version is 11.3.0 and lower, replace the soundhax file on the root of your SD card with the one from [this site](http://soundhax.saru.moe/frankenfirm.html?crash)
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
 <summary><u>"Could not play"</u></summary>
 
-You have the wrong Soundhax file for your console and region, or your console is incompatible with Soundhax. In the latter case, your course of action will determine on what version your 3DS is currently on. Join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for assistance.
+You have the wrong Soundhax file for your console, region and version. Double-check that you entered all the info correctly into the [Soundhax website](https://soundhax.com/), redownload your soundhax file, and replace it with the one on the root of your SD card.
+
+If the above doesn't fix your issue, and you are using an Old 3DS / Old 3DS XL / Old 2DS, you may be encountering a specific issue involving cartridge updates. Take another look at the system version. If the last number, e.g. 0 in 11.3.0-**0**U, is 3 and lower and the system version is 11.3.0 and lower, replace the soundhax file on the root of your SD card with the one from [this site](http://soundhax.saru.moe/frankenfirm.html?unplayable)
```

Comment:
see other review referencing this text
Link to safecerthax here if troubleshooting fail?